.github/workflows/ram_usage.yml

```diff
@@ -0,0 +1,94 @@
+name: SeqAn3 RAM-Usage
+
+on:
+  # Enables a manual trigger, may run on any branch
+  workflow_dispatch:
+    inputs:
+      compiler:
+        description: 'GCC version'
+        required: true
+        default: '11'
+      cxx_flags:
+        descriptions: 'CXXFLAGS to pass to compiler'
+
+env:
+  CMAKE_VERSION: 3.8.2
+  SEQAN3_NO_VERSION_CHECK: 1
+  TZ: Europe/Berlin
+
+defaults:
+  run:
+    shell: bash -exo pipefail {0}
+
+jobs:
+  build:
+    name: RAM-Usage gcc${{ github.event.inputs.compiler }}
+    runs-on: ubuntu-20.04
+    timeout-minutes: 300
+    steps:
+      - name: Checkout SeqAn3
+        uses: actions/checkout@v2
+        with:
+          path: seqan3
+          submodules: true
+
+      - name: Checkout SeqAn2
+        uses: actions/checkout@v2
+        with:
+          repository: seqan/seqan
+          ref: develop
+          path: seqan3/submodules/seqan
+
+      - name: Configure APT
+        continue-on-error: true
+        run: bash ./seqan3/.github/workflows/scripts/configure_apt.sh
+
+      - name: Install CMake
+        run: bash ./seqan3/.github/workflows/scripts/install_cmake.sh
+
+      - name: Install compiler g++-${{ github.event.inputs.compiler }}
+        run: sudo apt-get install --yes g++-${{ github.event.inputs.compiler }}
+
+      - name: Create gcc wrapper
+        run: |
+          echo -e '#!/bin/bash\n{ /usr/bin/time -v /usr/bin/g++-${{ github.event.inputs.compiler }} "$@"; }' > g++.sh
+          chmod +x g++.sh
+
+      - name: Configure tests
+        env:
+          CXX: ${{ github.workspace }}/g++.sh
+          CC: gcc-${{ github.event.inputs.compiler }}
+        run: |
+          mkdir seqan3-build
+          cd seqan3-build
+          cmake ../seqan3/test/unit -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="${{ github.event.inputs.cxx_flags }}" -DSEQAN3_VERBOSE_TESTS=OFF
+
+      - name: Update gcc wrapper
+        run: echo -e '#!/bin/bash\nFILE=$(mktemp ram_usage.XXXXXXXX)\n{ /usr/bin/time -v /usr/bin/g++-${{ github.event.inputs.compiler }} "$@" 2> $FILE; }' > g++.sh
```

Comment:
Huh? Wouldn't the configuration not work otherwise?

Reply:
Will fail to configure the `{i,o}stream_test` in `contrib`, I haven't looked into that.

Reply:
I needed to handle `-v` for zlib/bzip2 detection